### ISSUE TITLE
Can't start rapns daemon

### ISSUE THREAD

**Issue by ghost** [ileitch/rapns#37]
Getting this error when attempting to start rapns daemon. Already did generation and migration. Using ruby 1.8.7-p330 and rails 3.0.7. The app does not encounter this error when run with `rails server`, etc.

Not sure if this is a bug in rapns, but hoping someone might know the cause.

```
david@mac:~/Code/web/myapp$ bundle exec rapns -f
/Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/mysql-2.8.1/lib/mysql_api.bundle: Mysql is not a class (TypeError)
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:239:in `require'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:225:in `load_dependency'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:596:in `new_constants_in'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:225:in `load_dependency'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:239:in `require'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/mysql-2.8.1/lib/mysql.rb:3
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@global/gems/bundler-1.0.21/lib/bundler/runtime.rb:68:in `require'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@global/gems/bundler-1.0.21/lib/bundler/runtime.rb:68:in `require'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@global/gems/bundler-1.0.21/lib/bundler/runtime.rb:66:in `each'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@global/gems/bundler-1.0.21/lib/bundler/runtime.rb:66:in `require'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@global/gems/bundler-1.0.21/lib/bundler/runtime.rb:55:in `each'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@global/gems/bundler-1.0.21/lib/bundler/runtime.rb:55:in `require'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@global/gems/bundler-1.0.21/lib/bundler.rb:122:in `require'
        from /Users/david/Code/web/myapp/config/application.rb:7
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:239:in `require'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:239:in `require'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:225:in `load_dependency'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:596:in `new_constants_in'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:225:in `load_dependency'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:239:in `require'
        from ./config/environment.rb:2
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:235:in `load'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:235:in `load'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:225:in `load_dependency'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:596:in `new_constants_in'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:225:in `load_dependency'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/activesupport-3.0.7/lib/active_support/dependencies.rb:235:in `load'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/gems/rapns-1.0.4/bin/rapns:24
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/bin/rapns:19:in `load'
        from /Users/david/.rvm/gems/ruby-1.8.7-p330@ql_myapp/bin/rapns:19
```

environment.rb:

```
# Load the rails application
require File.expand_path('../application', __FILE__) # <---- line 2
require 'rest_client'

# ...
```

application.rb:

```
require File.expand_path('../boot', __FILE__)

require 'rails/all'

# If you have a Gemfile, require the gems listed there, including any gems
# you've limited to :test, :development, or :production.
Bundler.require(:default, Rails.env) if defined?(Bundler) # <---- line 7

# ...
```

Gemfile:

```
source 'http://rubygems.org'

gem 'rails', '3.0.7'

gem 'mysql'

gem 'json'

gem 'yaml_db'

gem 'engineyard'

gem 'authlogic'

gem 'paperclip'

gem 'mime-types', :require => 'mime/types'

gem 'aws-s3'

gem 'cancan'

gem 'composite_primary_keys', '=3.1.0'

gem 'nokogiri'

gem 'savon'

gem 'rapns'
```

Gemfile.lock:

```
GEM
  remote: http://rubygems.org/
  specs:
    abstract (1.0.0)
    actionmailer (3.0.7)
      actionpack (= 3.0.7)
      mail (~> 2.2.15)
    actionpack (3.0.7)
      activemodel (= 3.0.7)
      activesupport (= 3.0.7)
      builder (~> 2.1.2)
      erubis (~> 2.6.6)
      i18n (~> 0.5.0)
      rack (~> 1.2.1)
      rack-mount (~> 0.6.14)
      rack-test (~> 0.5.7)
      tzinfo (~> 0.3.23)
    activemodel (3.0.7)
      activesupport (= 3.0.7)
      builder (~> 2.1.2)
      i18n (~> 0.5.0)
    activerecord (3.0.7)
      activemodel (= 3.0.7)
      activesupport (= 3.0.7)
      arel (~> 2.0.2)
      tzinfo (~> 0.3.23)
    activeresource (3.0.7)
      activemodel (= 3.0.7)
      activesupport (= 3.0.7)
    activesupport (3.0.7)
    addressable (2.2.6)
    arel (2.0.9)
    authlogic (3.0.2)
      activerecord (~> 3.0.7)
      activerecord (~> 3.0.7)
    aws-s3 (0.6.2)
      builder
      mime-types
      xml-simple
    builder (2.1.2)
    cancan (1.6.4)
    composite_primary_keys (3.1.0)
      activerecord (>= 3.0.3)
    engineyard (1.3.33)
      engineyard-serverside-adapter (= 1.4.16)
      escape (~> 0.0.4)
      highline (~> 1.6.1)
      json_pure
      launchy (= 2.0.5)
      net-ssh (~> 2.1.0)
      rest-client (~> 1.6.0)
      thor (~> 0.14.6)
    engineyard-serverside-adapter (1.4.16)
      escape (~> 0.0.4)
      json_pure
    erubis (2.6.6)
      abstract (>= 1.0.0)
    escape (0.0.4)
    gyoku (0.4.3)
      builder (>= 2.1.2)
    highline (1.6.2)
    httpi (0.9.3)
      ntlm-http (>= 0.1.1)
      rack
    i18n (0.5.0)
    json (1.5.1)
    json_pure (1.6.1)
    launchy (2.0.5)
      addressable (~> 2.2.6)
    mail (2.2.19)
      activesupport (>= 2.3.6)
      i18n (>= 0.4.0)
      mime-types (~> 1.16)
      treetop (~> 1.4.8)
    mime-types (1.16)
    mysql (2.8.1)
    net-ssh (2.1.4)
    nokogiri (1.4.4)
    nori (0.2.0)
    ntlm-http (0.1.1)
    paperclip (2.3.11)
      activerecord (>= 2.3.0)
      activesupport (>= 2.3.2)
    polyglot (0.3.1)
    rack (1.2.2)
    rack-mount (0.6.14)
      rack (>= 1.0.0)
    rack-test (0.5.7)
      rack (>= 1.0)
    rails (3.0.7)
      actionmailer (= 3.0.7)
      actionpack (= 3.0.7)
      activerecord (= 3.0.7)
      activeresource (= 3.0.7)
      activesupport (= 3.0.7)
      bundler (~> 1.0)
      railties (= 3.0.7)
    railties (3.0.7)
      actionpack (= 3.0.7)
      activesupport (= 3.0.7)
      rake (>= 0.8.7)
      thor (~> 0.14.4)
    rake (0.8.7)
    rapns (1.0.4)
    rest-client (1.6.7)
      mime-types (>= 1.16)
    savon (0.9.2)
      builder (>= 2.1.2)
      gyoku (>= 0.4.0)
      httpi (>= 0.7.8)
      nokogiri (>= 1.4.0)
      nori (>= 0.2.0)
    thor (0.14.6)
    treetop (1.4.9)
      polyglot (>= 0.3.1)
    tzinfo (0.3.27)
    xml-simple (1.0.15)
    yaml_db (0.2.1)

PLATFORMS
  ruby

DEPENDENCIES
  authlogic
  aws-s3
  cancan
  composite_primary_keys (= 3.1.0)
  engineyard
  json
  mime-types
  mysql
  nokogiri
  paperclip
  rails (= 3.0.7)
  rapns
  savon
  yaml_db
```

**Comment by ileitch** [comment]
I've just released 1.0.5 which should fix this issue.

I _highly_ recommend you use the 'mysql2' gem instead, 'mysql' is no longer maintained.

**Comment by ghost** [comment]
Awesome, thanks!
